sdk/storage/azure-storage-file-datalake/pom.xml

```diff
@@ -55,7 +55,7 @@
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-storage-blob</artifactId>
-      <version>12.0.0</version> <!-- {x-version-update;com.azure:azure-storage-blob;current} -->
+      <version>12.0.0</version> <!-- {x-version-update;com.azure:azure-storage-blob;dependency} -->
```

Comment:
Should this be based on the released versions of Core and Blobs or do we want to update them back to 1.1.0-preview.1 and 12.1.0-preview.1?

If updating them back to the source code versions there will be import changes that need to be made, the inverse of #6210.